internal/tableland/impl/mesa.go

```diff
@@ -35,150 +36,131 @@ func NewTablelandMesa(
 
 // ValidateCreateTable allows to validate a CREATE TABLE statement and also return the structure hash of it.
 // This RPC method is stateless.
-func (t *TablelandMesa) ValidateCreateTable(
-	ctx context.Context,
-	req tableland.ValidateCreateTableRequest,
-) (tableland.ValidateCreateTableResponse, error) {
+func (t *TablelandMesa) ValidateCreateTable(ctx context.Context, statement string) (string, error) {
```

Comment:
Obviously some real changes in here, but will be easier to comment on in the interface declaration for `tableland.Tableland`.